src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -19,30 +22,82 @@ public class DeleteCommand extends Command {
     public static final String COMMAND_WORD = "delete";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Deletes the person identified by the index number used in the displayed person list.\n"
-            + "Parameters: INDEX (must be a positive integer)\n"
-            + "Example: " + COMMAND_WORD + " 1";
+            + ": Deletes the person(s) identified by the index number(s) used in the displayed person list.\n"
+            + "Parameters: any number of INDEX (must be a positive integer) separated by spaces\n"
+            + "Example: " + COMMAND_WORD + " 1 2";
 
     public static final String MESSAGE_DELETE_PERSON_SUCCESS = "Deleted Person: %1$s";
 
-    private final Index targetIndex;
+    public static final String MESSAGE_DELETE_PEOPLE_SUCCESS = "Deleted: ";
 
+    public static final String MESSAGE_INVALID_INDICES = "\nThe remaining inputs were invalid.";
+
+    private final ArrayList<Index> targetIndices;
+
+    private final boolean hasInvalidArgs;
+
+    /**
+     * Instantiates a DeleteCommand for deleting a single index assuming
+     * no invalid arguments inputted by user.
+     *
+     * @param targetIndex
+     */
     public DeleteCommand(Index targetIndex) {
-        this.targetIndex = targetIndex;
+        this.targetIndices = new ArrayList<Index>(Arrays.asList(targetIndex));
+        this.hasInvalidArgs = false;
+    }
+
+    /**
+     * Instantiates a DeleteCommand for deleting multiple indices assuming
+     * no invalid arguments inputted by user.
+     * @param targetIndices
```

Comment:
Very minor nit: might want to put a line break here between the description and the `@param` for consistency!